ckanext/maillog/cli.py

```diff
@@ -61,8 +83,9 @@ def send_logs(cleanup):
 
         smtp.sendmail(from_addr, [to_addr], msg.as_string())
 
-    click.echo(f"Sent {CKAN_MAILLOG_DIGEST_LOG_PATH} to {to_addr} via {smtp_server}")
+    click.echo(f"Sent {digest_log_path} to {to_addr} via {smtp_server}")
 
     if cleanup:
-        click.echo(f"Cleaning up {CKAN_MAILLOG_DIGEST_LOG_PATH}")
-        open(CKAN_MAILLOG_DIGEST_LOG_PATH, "w").close()
\ No newline at end of file
+        click.echo(f"Cleaning up {digest_log_path}")
+        for path in files:
+            open(path, "w").close()
```

Comment:
This overwrites the contents of the files, but leaves the empty files there. `os.path.isfile(f)` will still return `True` in this case, so I _think_ we will continue to send the empty files in the digest emails.

I might well be missing something to do with the log rotation that means this is not a problem, but if not, could we delete the files with `os.remove()` instead?

Reply:
Good point, I haven't put into consideration, yet! Will check that out, later to see, what makes most sense, thanks a lot!